TextInputKit/TextInputKit/Code/SpecializedTextInput/CardNumber/BankCardNumberUtils.swift

```diff
@@ -10,119 +10,64 @@ import Foundation
 
 struct BankCardNumberUtils {
 
-    struct IinRangeInfo {
-
-        let range: CountableClosedRange<Int>
-
-        let cardBrand: BankCardBrand
-
-        let maxCardNumberLength: Int
+    private init() {}
 
-        fileprivate init(_ range: CountableClosedRange<Int>, _ cardBrand: BankCardBrand, _ maxCardNumberLength: Int) {
-            self.range = range
-            self.cardBrand = cardBrand
-            self.maxCardNumberLength = maxCardNumberLength
-        }
+}
 
-    }
+extension BankCardNumberUtils {
```

Comment:
What reason for you break extension into a multiple ones?
I would use"// MARK - Section name".
Reducing number of extensions helps reduces compilation time.